resources/lang/vi/auth.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
```

Comment:
Sao lang/vi/... mà message toàn tiếng anh thế này em.
Em dowload lại file vi ở link git này nhé:
https://github.com/dinhquochan/laravel-vietnamese-language/tree/master/vi